.circleci/config.yml

```diff
@@ -400,13 +406,35 @@ jobs:
             IMPLEMENTATION_COVERAGE_FILE=scripts/implementation_coverage_full.csv \
             SOURCE_TYPE=community \
             python -m scripts.tinybird.upload_raw_test_metrics_and_coverage
-      - store_artifacts:
-          path: parity_metrics/
       - run:
-          name: store acceptance parity metrics
+          name: Create Coverage Diff (Code Coverage)
+          # pycobertura diff will return with exit code 0-3 -> we currently expect 2 (2: the changes worsened the overall coverage),
+          # but we still want cirecleci to continue with the tasks, so we return 0 always
           command: |
-            mkdir acceptance_parity_metrics
-            mv target/metric_reports/metric-report*acceptance* acceptance_parity_metrics/
+            source .venv/bin/activate
+            pip install pycobertura
+            coverage xml --data-file=.coverage -o all.coverage.report.xml --include="localstack/services/*/**" --omit="*/**/__init__.py"
+            coverage xml --data-file=.coverage.acceptance -o acceptance.coverage.report.xml --include="localstack/services/*/**"  --omit="*/**/__init__.py"
+            pycobertura show --format html -s localstack/ acceptance.coverage.report.xml -o coverage-acceptance.html
+            bash -c "pycobertura diff --format html -s1 localstack/ -s2 localstack/ all.coverage.report.xml acceptance.coverage.report.xml -o coverage-diff.html; if [[ \$? -eq 0 ]] ; then exit 1 ; else exit 0 ; fi"
```

Comment:
q: Here you return 1 in case 0, and 1 in case of anything else, so the exit code 0 seems to be a problem. Why is that? If you basically just want to completely ignore errors on this line, you could just pipe it to true:
https://support.circleci.com/hc/en-us/articles/360057590591-How-to-ignore-a-failure-in-a-step-

Reply:
I added a comment about it in line 411 above the `command`.
Basically for the current set we most likely will see exit code 2. 
From the docs:
```
Upon exit, the diff command may return various exit codes:

    0: all changes are covered, no new uncovered statements have been introduced
    1: some exception occurred (likely due to inappropriate usage or a bug in pycobertura)
    2: the changes worsened the overall coverage
    3: the changes introduced uncovered statements but the overall coverage is still better than before
```
Basically I was looking for a way to ignore that expected return code, and this was the one suggested way of doing it :) 
It might make sense to return code 1 though `1: some exception occurred` 🤔 

So I guess we could change it to: `if [[ \$? -eq 1 ]] ; then exit 1 ; else exit 0 ; fi`? WDYT?

Reply:
Oh, in this case I would say yes, we need to fail the step in case of exit code == 1, all other cases seem to be fine.